addresses/mainnet.json

```diff
@@ -104,6 +106,44 @@
 				"weightedPoolFactory": "0x8E9aa87E45e92bad84D5F8DD1bff34Fb92637dE9",
 				"convergentCurvePoolFactory": "0xb7561f547F3207eDb42A6AfA42170Cd47ADD17BD"
 			}
+		],
+		"dai": [
+			{
+				"expiration": 1702000550,
```

Comment:
probably should delete this tranche since it's a no go

Reply:
Removed!